docs/_static/style.css

```diff
@@ -1,4 +1,4 @@
-@import url("theme.css");
+@import url("styles/furo.css");
```

Comment:
this is not really usable, for jupyter outputs of xarray/pandas dataframes:
![Screenshot_20240429_110047](https://github.com/Ouranosinc/xclim/assets/91751417/26a00147-6436-4f76-8c71-3bcfa2c0da18)

(what information is supposed to be there:
![Screenshot_20240429_110148](https://github.com/Ouranosinc/xclim/assets/91751417/b4393e42-c01a-45bf-8445-2b89e7c7b43d)
)

Reply:
Note that this issue only applies when using the "auto" theme, not the "dark" or "light" theme. Probably a jupyter issue.

Reply:
I suggest we just disallow the "auto" theme altogether, if possible. as noted elsewhere the icon is not intuitive.

Reply:
Will look into this.

Reply:
See: https://github.com/pradyunsg/furo/discussions/790

Reply:
Existing issue specific to `xarray` objects: https://github.com/pydata/xarray/issues/8097#issuecomment-1688268711.